### PR TITLE
disable staking on farms

### PR DESCRIPTION
**Pull request by gbonumore**
##### Description
Removed the staking function and disabled the buttons to stake

### CODE REVIEW

File: src/pages/StakingPageSingle.svelte [hunk 1]
```diff
@@ -319,79 +319,12 @@
...
-            displayNotification({
-              message: Errors.USER_REFUSE_EXIT_FEES.message,
-              type: "hint",
-            });
-            return { ...Errors.USER_REFUSE_EXIT_FEES, context: {fee: data.exitFeePercentage}};
-          }
-        }
-      }
-
       return false;
```
Comment (jordaniza) [inline src/pages/StakingPageSingle.svelte]: I am assuming this final `return false` is the base case, so just deleting the stake logic will resolve to an unknown action and therefore false.

File: src/pages/StakingPageSingle.svelte [hunk 2]
```diff
@@ -445,7 +378,7 @@ metadata={{
               <div class="flex items-center justify-between">
                 <div class="flex nowrap intems-center p-1 font-thin">Amount to Stake</div>
                 <div class="right text-white font-bold text-xs py-1px text-center align-right float-right rounded">
-                  <button on:click={() => { stakeAmount = formatEther(data.userTokenBalance)}} class="oven-withdraw-button">100%</button>
+                  <button disabled on:click={() => { stakeAmount = formatEther(data.userTokenBalance)}} class="oven-withdraw-button">100%</button>
```
Comment (jordaniza) [inline src/pages/StakingPageSingle.svelte]: Why have the button here in the first place? If it's permanently disabled why not have a UI element that indicates staking is perma-disabled

File: src/pages/StakingPageSingle.svelte [hunk 3]
```diff
@@ -473,23 +407,8 @@ metadata={{
...
-                {:else}
-                  <button on:click={stake} class="btn clear stake-button mt-10px rounded-20px p-15px w-100pcborder-white">Stake</button>
-                {/if}
-              {:else}
-                <button disabled class="btn clear stake-button mt-10px rounded-20px p-15px w-100pc">Enter an amount</button>  
-              {/if}
-            {/if}
-            
-            
+              <button disabled class="btn clear stake-button mt-10px rounded-20px p-15px w-100pc">Deposits are disabled</button>
```
Comment (jordaniza) [inline src/pages/StakingPageSingle.svelte]: Same question

**Review by jordaniza — COMMENTED** [review]
No explicit changes requested (makes sense) just some quick questions from you then will approve

File: src/pages/StakingPageSingle.svelte (hunk 1 above)
Comment (gbonumore) [reply src/pages/StakingPageSingle.svelte]: every reference to action stake was removed from the codebase also

File: src/pages/StakingPageSingle.svelte (hunk 2 above)
Comment (gbonumore) [reply src/pages/StakingPageSingle.svelte]: That's why the button text was changed

**Review by jordaniza — APPROVED** [review]

**Review by Alexintosh — APPROVED** [review]

**Pull request merged**